test/unit/resources/etc_libuser.rb

```diff
@@ -0,0 +1,48 @@
+# encoding: utf-8
+# author: Thomas Cate
```

Comment:
I don't think this is you... did Thomas Cate take part in the writing of these tests?

Reply:
No it is not. This should be fixed now.